Updating README and plugin setup to better describe control parameter (#9)
biocore/q2-katharoseq: README.md

    --m-positive-control-column-file example/fmp_metadata.tsv \
    --m-positive-control-column-column control_rct \
    --m-cell-count-column-file example/fmp_metadata.tsv \
    --m-cell-count-column-column control_cell_into_extraction \
    --p-control classic \
    --o-visualization result_fmp_example.qzv
```
### Description of parameters:
If you type `qiime katharoseq read-count-threshold` into the command line, the following information will be displayed.

- `table`:  A qiime FeatureTable collapsed to the genus level (level 6) that contains the control samples.
- `threshold`: Threshold to use in calculating minimum frequency. Must be int in 0 .. 100.
- `positive-control-value`: The value in the control column that demarks which samples are the positive controls.
- `positive-control-column-file`: .tsv metadata filename
- `positive-control-column-column`: The column name for the column in the sample metadata that describes which samples are and are not controls.
- 
- `cell-count-column-file`: .tsv metadata filename (probably the same as `positive-control-column-file`)
- `cell-count-column-column`: Column name in metadata that contains cell count information

> - `control`: Community of controls used. IMPORTANT: please see the below section on possible control communities. If there is a species in your control community that is not in any of the provided builtin options, please contact the authors of this tool. Custom control communities are not yet supported but will be in the future.

Reviewer: ```suggestion
- `control`: Community of controls used. IMPORTANT: please see the below section on possible control communities. If there is a species in your control community that is not in any of the provided builtin options, please open an issue. Custom control communities are not yet supported but will be in the future.
```

    --o-visualization biomass_plot_fmp
```

To get a description of parameters type `qiime katharoseq biomass-plot` into the command line.

## Control Communities for read-count-threshold

> If your control community is a subset of any of these, use that category, however if your control community contains speciese that are not in that group, or if they are different altogether, please contact dsperry@ucsd.edu. Support for custom controls will be included soon.

Reviewer: ```suggestion
If your control community is a subset of any of these, use that category, however if your control community contains species that are not in that group, or if they are different altogether, please open an issue. Support for custom controls will be included soon.
```
